config/routes.rb

```diff
@@ -1639,6 +1639,7 @@
         alert_profile_edit
         alert_profile_field_changed
         button
+        clear_search_text
```

Comment:
@h-kataria This BZ was for these particular controllers, we can/should fix other controllers, but probably in separate PRs?